src/scss/yellow/components/_form.scss

```diff
@@ -143,12 +154,18 @@
       width: 100%;
     }
 
-    &:hover,
-    &:focus {
-      @extend %focus-style;
+    &:hover {
+      background-color: $color-interface-shade-light;
+      border: 1px solid $color-interface-content-extra-dark;
       outline: none;
     }
 
+    &:focus {
+      background-color: $color-interface-content-light;
+      border-color: $color-brand-first-extra-dark;
+      color: $color-interface-content-dark;
+    }
+
     &--filled {
       background-color: $color-interface-content-light;
       border: 1px solid get-color((color: rastaGray, opacity: .1));
```

Comment:
This color is not used in category page. When we get a page that uses it, we will change to the correct color.